azure-cli/ad/Add-GroupToServicePrincipal.ps1

```diff
@@ -0,0 +1,33 @@
+function Add-GroupToServicePrincipal(
+    [Parameter(Mandatory = $true)]
+    [EnvironmentConfig] $environmentConfig,
```

Comment:
Use pascal case for parameter name

Reply:
I was not aware that we wanted to use pascal case and when I look at other snippets it seems that I am not the only one who defaulted to camel case. Is this something that has been discussed on a CodeJam that I missed?

Reply:
I have been assigned "Responsible Member" on this repository. I have put in a lot of effort to researching best practices and found that pascal case is "strongly encouraged" for parameter names.
https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.2#use-pascal-case-for-parameter-names

There is _alot_ of inconsistencies and style issues throughout this repository and I intend to fix up everything, eventually.
If you think that this rule is bad, I guess we should discuss on this on a code jam.

I completely agree with you that we should have a coding rules readme, it is something that I has already discussed with @perkops  and @davidkallesen . But haven't found the time to create yet. 
When we get this repository to a working state, we are going to add PSScriptAnalyzer to GitHub actions which will automatically find these inconsistencies in new PRs.

Reply:
I do not think the rule is a bad idea, I was just curious about where it came from. I think Microsoft is a pretty reasonable source so I will change these to pascal case.